crawl-ref/source/mon-transit.cc

```diff
@@ -142,7 +144,16 @@ void place_followers()
 static bool _place_lost_monster(follower &f)
 {
     dprf("Placing lost one: %s", f.mons.name(DESC_PLAIN, true).c_str());
-    return f.place(false);
+    if (f.place(false))
```

Comment:
Consider swapping the order of these two clauses, so that the error handling is simple and up-front, and then the main body of the function can be read all at once and less indented.